### PR TITLE
cd: update Spanish translation

### PR DESCRIPTION
**Pull request by Magrid0**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.es/windows/cd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`cd`
+
+- Mover al directorio superior:
+
+`cd ..`
+
+- Mover a un directorio específico en el mismo disco:
+
+`cd {{ruta/al/directorio}}`
```
Comment (kbdharun) [inline pages.es/windows/cd.md:16]:
```suggestion
`cd {{ruta\al\directorio}}`
```

File: pages.es/windows/cd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Mover a un directorio específico en otro disco:
+
+`cd /d {{C}}:{{ruta/al/directorio}}`
```
Comment (kbdharun) [inline pages.es/windows/cd.md:20]:
```suggestion
`cd /d {{C}}:{{ruta\al\directorio}}`
```

File: pages.es/windows/cd.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Muestra información sobre el directorio actual y le permite moverse a otro.
+> Más informaciones: <https://learn.microsoft.com/windows-server/administration/windows-commands/cd>.
+
+- Muestra la ruta (path) del directorio actual:
+
+`cd`
+
```
Comment (kbdharun) [inline pages.es/windows/cd.md:9]: Seems like this page is missing an example from the [original page](https://github.com/tldr-pages/tldr/blob/main/pages/windows/cd.md).

```suggestion

- Ir a la raíz de la unidad actual:

`cd \`

```
Feel free to reword the suggestion I made using Google translate.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**